src/pages/index.en.js

```diff
@@ -81,24 +86,27 @@ const IndexPage = ({ location }) => (
             <ResponsiveImage src={authorizationImage} alt="Illustratie van desktop computer en mobiele applictie met een slot" />
           </Box>
           <Box width={3/5}>
-            <h2>TODO: Demodam value/benefit statement 3</h2>
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            <h2>Demoman helps to improve Common Ground</h2>
+            <p>Together we set the standards for the components, regarding for example security user friendliness and (coding) standards. We learn from each other what the best practices are. The ultimate goal is to show all Demodam components in an app store like environment, comparable to that on your mobile phone, where you can install components and applications with a single click. 
+            </p>
           </Box>
         </Flex>
       </Section>
 
       <Section>
         <Flex>
           <Box width={3/5}>
-            <h2>TODO: Demodam value/benefit statement 4</h2>
-            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+            <h2>With Demodam you can innovate</h2>
+            <p>Soon a solid base will exist, which will serve as a foundation for future work. As an example: say you want to develop a service spanning different governmental departments, for example a customer journey where a civilian wants to both register their child’s birth (with the municipality) and apply for child allowances (with the tax authorities) in one smooth process. Currently this is very difficult to execute, both technically and organizationally, but with Demodam soon a case system, personal registration system and NLX will be ready to make sure you can get underway fast.
+            </p>
           </Box>
           <Box width={2/5}>
             <ResponsiveImage src={dashboardImage} alt="Illustratie van persoon met tablet waarop grafieken getoond worden" />
           </Box>
         </Flex>
       </Section>
 
+<!---
```

Comment:
It looks like the build is breaking here. I am not sure, but can it be because it is a html comment in a .js file?

Reply:
Looks like it is the commenting indeed. I commented that section out because it was basically unused (the dutch page listed 4 benefits, but there were 5 boxes). The part that is between the comments can be left out without any problems I suppose.

Reply:
Or you could try the /*  */ syntax and see if it passes the tests.

Reply:
I removed the section, let's see how it goes now